code/game/machinery/autolathe.dm

```diff
@@ -77,6 +79,15 @@
 	materials.retrieve_all()
 	return ..()
 
+/obj/machinery/autolathe/proc/on_tool_attack(datum/source, atom/tool, mob/user)
+	var/obj/item/I = tool
+	if(!istype(I))
+		return
+	// Allows screwdrivers to be recycled on harm intent
+	if(I.tool_behaviour == TOOL_SCREWDRIVER && user.a_intent == INTENT_HARM)
+		return COMPONENT_CANCEL_TOOLACT
+	return
```

Comment:
Also this hanging return can also be deleted
```suggestion
```

Reply:
ah okay so . defaults to null